composer.json

```
@@ -61,6 +61,7 @@
         "drupal/token": "1.9",
         "drupal/views_bootstrap": "4.3",
         "drupal/viewsreference": "2.0-beta2",
+        "drupal/webform": "6.0",
```

I think we should probably use the full latest version number:
```suggestion
        "drupal/webform": "6.0.1",
```